index.html

```diff
@@ -286,7 +461,57 @@ <h5>LET'S CONNECT</h5>
 
             </ul>
 
-        </div>
+        </article>
+
+    </section>
+
+    <section class="bg_color7">
+
+        <article class="contact_section">
```

Comment:
You are almost there with the design. There are some small details remaining to achieve perfection. 🤓️
- [ ] Kindly check if the correct background image for this section is being used.
- [ ] Kindly make sure the paragraph and the `form` element are positioned in the center of the screen. You can give these elements a `max-width` property to achieve this alignment.

- Your Design
![Screenshot from 2022-12-08 20-33-11](https://user-images.githubusercontent.com/17243325/206537507-0a7acdc2-4849-45a1-a33a-2c86a5f9dfd5.png)

- Figma Design
![Screenshot from 2022-12-08 20-32-35](https://user-images.githubusercontent.com/17243325/206537584-1b44d7e1-c294-42ac-98d7-7e25c9055b64.png)